src/index.ts

```diff
@@ -141,10 +141,11 @@ export default class Combobox {
     for (const el of this.list.querySelectorAll('[aria-selected="true"]')) {
       el.removeAttribute('aria-selected')
```

Comment:
I think we should probably also remove `data-` attribute that indicates the 'active' default option here:

```suggestion
      el.removeAttribute('aria-selected')
      el.removeAttribute('data-combobox-option-default')
```

Reply:
Addressed in https://github.com/github/combobox-nav/pull/82/commits/7182ef587de85db41fc1213579ff63f8d4eab6a2 🙇